solidity/contracts/StaticOracle.sol

```diff
@@ -56,8 +57,7 @@ contract StaticOracle is IStaticOracle {
     address _quoteToken,
     uint32 _period
   ) external view override returns (uint256 _quoteAmount, address[] memory _queriedPools) {
-    _queriedPools = _getQueryablePoolsForTiers(_baseToken, _quoteToken, _period);
-    _quoteAmount = _quote(_baseAmount, _baseToken, _quoteToken, _queriedPools, _period);
+    quoteAllAvailablePoolsWithOffsettedTimePeriod(_baseAmount, _baseToken, _quoteToken, _period, 0);
```

Comment:
You are missing the `return` here (and on the other functions too)

Reply:
OH! 😨

Reply:
sorry didn't run `yarn test:all` as it required to have lots of RPCs, fixed now!